public_html/wp-content/plugins/camptix-attendee-survey/includes/page.php

```diff
@@ -0,0 +1,311 @@
+<?php
+/**
+ * Adds a page which contains the survey.
+ */
+
+namespace CampTix\AttendeeSurvey\Page;
+
+defined( 'WPINC' ) || die();
+
+use function CampTix\AttendeeSurvey\{get_survey_page_id};
+
+add_filter( 'pre_trash_post', __NAMESPACE__ . '\prevent_deletion', 10, 2 );
+add_filter( 'pre_delete_post', __NAMESPACE__ . '\prevent_deletion', 10, 3 );
+
+/**
+ * Constants.
+ */
+const SURVEY_PAGE_ID = 'attendee_survey_page';
+
+
+/**
+ * Return the page ID.
+ *
+ * @return mixed
+ */
+function get_page_id() {
+	return get_option( SURVEY_PAGE_ID );
+}
+
+
+/**
+ * Prevent deletion of the Survey page, unless force_delete is true.
+ *
+ * @param bool|null $check Whether to go forward with trashing/deletion.
+ * @param WP_Post   $post  Post object.
+ * @param bool      $force_delete Whether to bypass the trash, set when deactivating the plugin to clean up.
+ */
+function prevent_deletion( $check, $post, $force_delete = false ) {
+	$survey_page = (int) get_option( SURVEY_PAGE_ID );
+
+	if ( $survey_page === $post->ID ) {
+		// Allow it, and delete the option if the page is force-deleted.
+		if ( $force_delete ) {
+			delete_option( SURVEY_PAGE_ID );
+			return $check;
+		}
+
+		return false;
+	}
+
+	return $check;
+}
+
+/**
+ * Get the content of the survey page.
+ *
+ * @return string
+ */
+function get_page_content() {
+	return <<<EOT
+	<!-- wp:paragraph -->
+	<p>If you recently attended a WordPress event, please take the survey below to help us make them even better in the future.</p>
+	<!-- /wp:paragraph -->
+
+	<!-- wp:jetpack/contact-form {"subject":"New feedback received from your website","to":"admin@wordcamp.test","style":{"spacing":{"padding":{"top":"16px","right":"16px","bottom":"16px","left":"16px"}}}} -->
```

Comment:
@iandunn What email address do you think this should go to?

Reply:
I'll just let jetpack send it to the admin:
https://github.com/Automattic/jetpack/blob/8b14bf5ca68fb1876c0f7ef7f1c60c5a93039f3a/projects/packages/forms/src/contact-form/class-contact-form.php#L82

Reply:
That makes sense 👍🏻